webapp/greenhouse_api.py

```diff
@@ -9,8 +13,8 @@
     fallback_cache_duration=300, file_cache_directory=".webcache"
 )
 
-
-base_url = "https://api.greenhouse.io/v1/boards/Canonical/jobs"
+base_url = "https://boards-api.greenhouse.io/v1/boards/Canonical/jobs"
+API_KEY = os.environ["GREENHOUSE_API_KEY"]
```

Comment:
Doing this here means the whole app will fail to run without this key - which is why the "test-site" CI test is failing, and it will also prevent demos from starting.

I suggested you do this at line 61 precisely because then you'd only see this error when you tried to load that specific page.

Personally I'm also not a big fan in general of keeping things in global constants when those things are used in only one or two places, so I'd prefer that you read the environment variable directly in the view function from a style perspective as well.